lib/rucio/common/config.py

```diff
@@ -709,6 +709,7 @@ def get_config_dirs() -> list[str]:
     Returns all available configuration directories in order:
     - $RUCIO_HOME/etc/
     - $VIRTUAL_ENV/etc/
+    - $CONDA_ENV/etc
```

Comment:
AFAIK `CONDA_ENV` is not a default environment variable - did you mean `CONDA_PREFIX` here?

Reply:
ops! the code actually is correct